ENIGMAsystem/SHELL/Graphics_Systems/None/NONEbackground.cpp

```diff
@@ -0,0 +1,23 @@
+/** Copyright (C) 2017 Faissal I. Bensefia
```

Comment:
This file doesn't do anything and can be deleted.

Reply:
Done